packages/gitbook/src/app/(site)/(content)/[[...pathname]]/page.tsx

```diff
@@ -77,20 +77,7 @@ export default async function Page(props: {
             {withFullPageCover && page.cover ? (
                 <PageCover as="full" page={page} cover={page.cover} context={contentRefContext} />
             ) : null}
-            <div className={tcls('flex', 'flex-row')}>
-                <PageBody
-                    space={space}
-                    pointer={contentPointer}
-                    contentTarget={contentTarget}
-                    customization={customization}
-                    context={contentRefContext}
-                    page={page}
-                    document={document}
-                    withPageFeedback={
-                        // Display the page feedback in the page footer if the aside is not visible
-                        withPageFeedback && !page.layout.outline
-                    }
-                />
+            <div className="flex flex-row-reverse">
```

Comment:
Can you add a comment to explain why? otherwise someone will simplify this and remove it